### PR TITLE
fix: too long description no longer makes motion status column compre…

### PR DESCRIPTION
**Pull request by joanna-pagepro**
https://tw.auroracreation.com/app/tasks/15692489

### CODE REVIEW

**Review by TomaszChmielPagepro — DISMISSED** [review]

**Review by arrenv — DISMISSED** [review]
@joanna-pagepro I went to test this, I entered a long title and it wrapped correctly, but then I entered a long description and it did the following before even creating the motion:

image

**Comment by joanna-pagepro** [comment]
@arrenv can you check if it's ok now?

**Review by TomaszChmielPagepro — DISMISSED** [review]

**Review by TomaszChmielPagepro — DISMISSED** [review]

**Review by melyndav — APPROVED** [review]
Looks good @joanna-pagepro, thanks for the work on this one.

File: src/components/v5/shared/RichText/RichText.tsx [hunk 1]
```diff
@@ -36,14 +36,14 @@ const RichText: FC<RichTextProps> = ({
...
-                className={clsx('sm:hover:text-blue-400', {
+                className={clsx('sm:hover:text-blue-400 w-full', {
                   'text-gray-900': characterCount,
                   'text-gray-400': !characterCount,
                 })}
               >
                 <span
                   className={clsx({
-                    'line-clamp-2 text-left':
+                    'line-clamp-2 text-left break-all':
```
Comment (jakubcolony) [inline src/components/v5/shared/RichText/RichText.tsx:46]: Would `break-words` work better here? `break-all` will not try to preserve whole words

**Review by jakubcolony — DISMISSED** [review]
LGTM 👍

**Review by bassgeta — APPROVED** [review]

**Review by jakubcolony — APPROVED** [review]

**Review by rdig — APPROVED** [review]

**Pull request merged**